pytest_randomly.py

```diff
@@ -82,6 +86,15 @@ def _reseed(config, offset=0):
             np_random.set_state(np_random_states[seed])
 
 
+def pytest_configure(config):
+    if config.option.randomly_repeat_last and config.cache.get('last_seed', None):
```

Comment:
To namespace the cache value (since it's all one big key-value store in pytest), I've changed this to `randomly_last_seed` - inline with the command-line options all starting with ``randomly``!